src/AddTodoForm.js

```diff
@@ -1,33 +1,35 @@
-import React, {useState} from 'react';
+import React, {useState} from 'react'; 
+import InputWithLabel from './InputWithLabel';
 
-<input name = 'title' />
 function AddTodoForm({onAddTodo}) {
     const [todoTitle, setTodoTitle] = useState('');
+    
     function handleTitleChange(event) {
         const newTodoTitle = event.target.value;
         setTodoTitle(newTodoTitle)
     }
     function handleAddTodo(event) {
+
         const newTodo = {
             title: todoTitle,
             id: Date.now()
           };
 
-        event.preventDefault()
```

Comment:
try adding this line back in along with the `useEffect` dependency change, and then I believe everything should be working correctly

Reply:
Will do, thank you!